master/getting-started/kubernetes/installation/aws.md

```diff
@@ -2,7 +2,7 @@
 title: Deploying Calico and Kubernetes on AWS
 ---
 
-These instructions allow you to set up a Kubernetes cluster with Calico networking on AWS using the [Calico CNI plugin][calico-cni]. This guide does not setup TLS between Kubernetes components or on the Kubernetes API.
+These instructions allow you to set up a Kubernetes cluster with Calico networking on AWS using the [Calico CNI plugin][cni-plugin]. This guide does not setup TLS between Kubernetes components or on the Kubernetes API.
```

Comment:
At the bottom of this doc there are a few links which can be referenced via `[]` tags in the context of the text.

We're not super consistent about its use.  It can be useful if you have really long links or links that are used a lot in one doc, but generally I prefer _not_ to use this because I find the links at the bottom get out of date and you end up with links that aren't used in the body of the doc any more :)